src/formatDate.js

```diff
@@ -9,6 +9,38 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
-}
+  const data = date.split(fromFormat[3]);
+
+  const object = {};
+
+  for (let i = 0; i < 3; i++) {
+    object[fromFormat[i]] = data[i];
+  }
+
+  if (object.YYYY) {
+    object.YY = object.YYYY.slice(2);
+  }
+
+  if (object.YY < 30) {
+    object.YYYY = '20' + `${object.YY}`;
+  } else {
+    object.YYYY = '19' + `${object.YY}`;
+  }
 
+  const objectKeys = Object.keys(object);
+  const objectValues = Object.values(object);
+
+  let result = [];
+
+  for (let i = 0; i < toFormat.length; i++) {
+    for (let j = 0; j < objectKeys.length; j++) {
+      if (toFormat[i] === objectKeys[j]) {
+        result.push(objectValues[j]);
+      }
+    }
+  }
```

Comment:
The nested loop here is not efficient. Consider using a single loop over the `toFormat` array and directly mapping the format keys to their corresponding values from the `object`. This will simplify the logic and improve performance.